README.md

```diff
@@ -17,8 +17,8 @@ Add this to your application's `shard.yml`:
 
 ```yaml
 dependencies:
-  amber_spec:
-    github: amberframework/amber-spec
+  garnet-spec:
```

Comment:
We should use `garnet_spec` here, we already have `jasper_helper`, `granite_orm` and `quartz_mailer`